qemu/tests/iozone_linux.py

```diff
@@ -0,0 +1,62 @@
+import re
+
+from virttest import error_context
+from virttest import utils_disk
+from virttest import utils_misc
+from provider.storage_benchmark import generate_instance
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Run IOzone for linux on a linux guest:
+    1) Log into a guest.
+    2) Execute the IOzone test.
+    :param test: kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    def _get_data_disks():
+        """ Get the data disks by serial or wwn options. """
+        disks = {}
+        for data_image in params["images"].split()[1:]:
+            extra_params = params.get("blk_extra_params_%s" % data_image, '')
+            match = re.search(r"(serial|wwn)=(\w+)", extra_params, re.M)
+            if match:
+                drive_id = match.group(2)
+            else:
+                continue
+            drive_path = utils_misc.get_linux_drive_path(session, drive_id)
+            if not drive_path:
+                test.error("Failed to get '%s' drive path" % data_image)
+            disks[drive_path[5:]] = data_image
+        return disks
+
+    iozone_test_dir = params.get('iozone_test_dir', '/home')
+    iozone_cmd_options = params['iozone_cmd_options']
+    iozone_timeout = float(params.get("iozone_timeout", 1800))
+    n_partitions = params.get('partitions_num', 1)
+    fstype = params.get('fstype', 'xfs')
+    labeltype = params.get('labeltype', utils_disk.PARTITION_TABLE_TYPE_GPT)
```

Comment:
@XueqiangWei could you confirm which default fstype and labeltype settings for linux basic iozone testing here? for iozone_window, the default remain use NTFS and MBR.

Reply:
default values are xfs and msdos for linux, but I think xfs and gpt are also ok here. We just focus on virtio storage in use.